src/haversine.jl

```diff
@@ -61,4 +62,4 @@ end
 
 spherical_angle(x, y) = SphericalAngle()(x, y)
 
-result_type(::Union{Haversine, SphericalAngle}, ::Type, ::Type) = Float64
```

Comment:
Is the `SphericalAngle` `result_type` taken care of elsewher?

Reply:
Oh crap, I didn't mean to remove the union; good catch :+1: